doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -0,0 +1,13 @@
+API Changelog 
+=============
+
+.. contents:: |toctitle|
+    :local:
+    :depth: 1
+
+6.0.0
+-----
+
+Changes
+~~~~~~~
+- **api/access/datafile**: When a null or invalid API Key is provided to download a public with this API call, it will result on a ``401`` error response. 
```

Comment:
```suggestion
- **/api/access/datafile**: When a null or invalid API Key is provided to download a public with this API call, it will result on a ``401`` error response. 
```

I think it might look a little nicer as `/api` instead of just `api`